### DISCUSSION TITLE
How can I create a slash command?

### DISCUSSION THREAD

**Discussion by Batuhan337tr** [discord-php/DiscordPHP#1130] [Q&A] [answered]
How can I create a / command? Does DiscordPHP support this? I want to create a command like the one in the attachment, but I couldn't find a solution in the documentation. Can you provide an example command that outputs '/ping pong'?

![test](https://github.com/discord-php/DiscordPHP/assets/47800213/abf59721-b2bd-437a-bd3e-5cc2a48f06a3)

**Comment by brunofunnie** [comment]
After created how to remove it?

**Reply by SQKo** [reply]
Added in the same wiki page

https://github.com/discord-php/DiscordPHP/wiki/Slash-Command#deleting-command

**Comment by shehi** [comment]
Read Discord API Docs and figured it out?

**Comment by valzargaming** [comment, marked as answer]
```php
$discord->application->commands->freshen()->then(function (GlobalCommandRepository $commands): void
{
    // if ($command = $commands->get('name', 'ping')) $commands->delete($command);
    if (! $commands->get('name', 'ping')) $commands->save(new Command($discord, [
        'name'        => 'ping',
        'description' => 'Replies with Pong!',
    ]));

    $discord->listenCommand('ping', fn (Interaction $interaction): PromiseInterface =>
        $interaction->respondWithMessage('Pong!', true)
    );
});
```
